src/screens/AdminPortal/PluginStore/UploadPluginModal.spec.tsx

```
@@ -136,9 +136,8 @@ describe('UploadPluginModal Component', () => {
 
   describe('File Upload', () => {
     it('should handle file selection and show file name', async () => {
-      const { validateAdminPluginZip } = await import(
-        'utils/adminPluginInstaller'
-      );
+      const { validateAdminPluginZip } =
+        await import('utils/adminPluginInstaller');
```

ESLint

[error] Replace `⏎········await·import('utils/adminPluginInstaller'` with `·await·import(⏎········'utils/adminPluginInstaller'⏎······`

(prettier/prettier)